src/test/java/com/sap/sgs/phosphor/fosstars/nvd/NVDTest.java

```diff
@@ -54,4 +54,60 @@ public void find() throws IOException {
       assertEquals(0, entries.size());
     }
   }
+
+  @Test
+  public void matcher() throws IOException {
+    TestNVD nvd = new TestNVD();
+    try (InputStream content = getClass().getResourceAsStream("NVD_matcher.json")) {
+      nvd.add("file.json", content);
+      nvd.parse();
+
+      List<NvdEntry> entries = nvd.search(
+          NvdEntryMatcher.entriesFor(new GitHubProject("spring-projects", "spring-framework")));
+      assertNotNull(entries);
+
+      assertEquals(20, entries.size());
+
+      entries = nvd.search(
+          NvdEntryMatcher.entriesFor(new GitHubProject("spring-projects", "spring-integration")));
+      assertNotNull(entries);
+      assertEquals(0, entries.size());
+
+      entries = nvd.search(NvdEntryMatcher
+          .entriesFor(new GitHubProject("spring-projects", "spring-framework-issues")));
+      assertNotNull(entries);
+      assertEquals(0, entries.size());
+
+      entries = nvd
+          .search(NvdEntryMatcher.entriesFor(new GitHubProject("spring-projects", "spring-boot")));
+      assertNotNull(entries);
+      assertEquals(0, entries.size());
+
+      entries = nvd.search(NvdEntryMatcher.entriesFor(new GitHubProject("FasterXML", "jackson")));
+      assertNotNull(entries);
+      assertEquals(0, entries.size());
+
+      entries = nvd.search(NvdEntryMatcher.entriesFor(new GitHubProject("netty", "netty")));
+      assertNotNull(entries);
+      assertEquals(3, entries.size());
+
+      entries =
+          nvd.search(NvdEntryMatcher.entriesFor(new GitHubProject("openssl", "openssl-book")));
+      assertNotNull(entries);
+      assertEquals(0, entries.size());
+
+      // The below test case tests for both cpe22Uri and cpe23Uri.
+      entries = nvd
+          .search(NvdEntryMatcher.entriesFor(new GitHubProject("FasterXML", "jackson-databind")));
+      assertNotNull(entries);
+      assertEquals(5, entries.size());
+
```

Comment:
Please remove this extra line.

Reply:
I have removed this. I am not sure why it is not getting updated.